package.json

```diff
@@ -184,8 +184,8 @@
     }
   },
   "engines": {
-    "node": ">=14.0.0",
-    "npm": ">=7.0.0"
+    "node": ">=18.0.0",
+    "npm": ">=8.0.0"
```

Comment:
Shouldn't this be 9?

Reply:
I guess yes, for precisely 18.0 seems it still comes with 8.x, newer come with 9.x

Reply:
Oh. I don't think 18.0 was LTS so I'd say requiring npm 9 is good.